src/api/stats/fantom/getSolidlyApys.js

```diff
@@ -0,0 +1,48 @@
+const { fantomWeb3: web3 } = require('../../../utils/web3');
+const { FANTOM_CHAIN_ID: chainId } = require('../../../constants');
+import { getEDecimals } from '../../../utils/getEDecimals';
+const { getSolidlyGaugeApys } = require('../common/getSolidlyGaugeApys');
+
+const volatilePools = require('../../../data/fantom/solidlyLpPools.json');
+
+import { addressBook } from '../../../../packages/address-book/address-book';
+const {
+  fantom: {
+    tokens: { SOLID },
+  },
+} = addressBook;
+
+const pools = [...volatilePools];
+const getSolidlyApys = async () => {
+  const gaugeApys = getSolidlyGaugeApys({
+    web3: web3,
+    chainId: chainId,
+    pools: pools,
+    oracleId: 'SOLID',
+    oracle: 'tokens',
+    decimals: getEDecimals(SOLID.decimals),
+    reward: SOLID.address,
+    boosted: false,
+    // log: true,
+  });
+
+  let apys = {};
```

Comment:
its already breaked down no need to repeat. just:
```js
const getSolidlyApys = async () =>
  getSolidlyGaugeApys({
    web3: web3,
    chainId: chainId,
    pools: pools,
    oracleId: 'SOLID',
    oracle: 'tokens',
    decimals: getEDecimals(SOLID.decimals),
    reward: SOLID.address,
    boosted: false,
    // log: true,
  });
```

can also consider to unify imports and requires or just drop imports and pass solid address and 1e18 as literals same you did with oracle